components/multi-platform-controller/staging-downstream/host-config.yaml

```diff
@@ -67,17 +93,6 @@ data:
   dynamic.linux-m8xlarge-arm64.max-instances: "10"
   dynamic.linux-m8xlarge-arm64.subnet-id: subnet-07597d1edafa2b9d3
 
-  dynamic.linux-amd64.type: aws
```

Comment:
Git remembers everything. IMO there no need to keep disabled/commented out code just so we can restore it.

Reply:
I leave it up to @hugares to decide if we need that

Reply:
We can maintain them but I am not convinced that we need. The other flavors (mxlarge, m2xlarge,...) are there to provide a range of machine with more resources than the default ones. In this case, the default amd64 being replaced had 2vCPU/*GiB RAM and our nodes in the cluster have 8vCPU/32GiB RAM.

Resource wise, there is no point of keeping it (in fact, we could get rid m-xlarge too)BUT, it might be useful if we want to preserve the capability to do  a remote build with amd64 on a VM that does not cost too much. @arewm Is this the reason you want to preserve it?

Reply:
Yes, that is why I wanted to keep it so that we can foster responsible consumption of resources.

Reply:
I agree then with original comment, let's maintain these as linux-mlarge/amd64

Reply:
We can add the mlarge option to enable migration in the future even if we do not introduce the breaking change of modifying the behavior of `linux/amd64` immediately.